bundle/tests/scorecard/config.yaml

```diff
@@ -0,0 +1,29 @@
+kind: Configuration
+apiversion: scorecard.operatorframework.io/v1alpha3
+metadata:
+  name: config
+
+stages:
+  - tests:
+      - image: quay.io/ytripath/cert-manager-operator-e2e-tests:latest
```

Comment:
This should be parametrized. Maybe Kustomize could help here?

Reply:
@slaskawi Would it be okay, if I added configs to generate this file when `make bundle` is run, using `kustomize`